assets/template/Cargo.toml_template

```diff
@@ -2,6 +2,7 @@
 name = "${package_name}"
 version = "0.1.0"
 edition = "2021"
+resolver = "2"
```

Comment:
I'm putting this comment here to explain why I made those changes to the Cargo.toml file template.

With this PR, the blueprint macro generates the test bindings for us. Those test bindings are feature gated behind the `test` feature. This allows us to:

* Be able to use the test bindings in tests.
* Not increase the size of the WASM since the bindings are feature gated.

So, we would like to test this crate with the `test` feature enabled to get the test bindings. This requires us to use the new feature resolver and to add this crate as a dev-dependency with the test feature enabled.